### DISCUSSION TITLE
What is exactly the ref that we are passing to the store?

### DISCUSSION THREAD

**Discussion by juanmaguitar** [WordPress/gutenberg#53020] [Interactivity API]
Is it equivalent to [Preact ref](https://href.li/?https://preactjs.com/tutorial/05-refs/)?

**Comment by DAreRodz** [comment]
Indeed. 🙂 It is basically the `current` attribute of a [Preact ref](https://href.li/?https://preactjs.com/tutorial/05-refs/) instance. The idea is to allow directives to access the DOM element that contains them.

The `ref` variable is not passed to the store, though; it is an argument that directives receive when executed.
